docs/index.mdx

```diff
@@ -1,10 +1,26 @@
 # MDX
 
+<div style={{ color: 'white', backgroundColor: 'black', padding: '24px 32px' }}>
+
+<a
+  style={{
+    color: 'white',
+    textDecoration: 'none',
+    fontWeight: 'bold',
+    fontSize: 32
+  }}
+  href="https://blacklivesmatters.carrd.co/"
+>
+  #BlackLivesMatter &rarr;
```

Comment:
with mdx 2 will we need an escape char here? to prevent hashtag from being interpreted as a heading?

```suggestion
  \#BlackLivesMatter &rarr;
```

Reply:
Headings don’t form without the space in commonmark, so that shouldn’t be needed.
Is some tool complaining here or was this your hunch?

Reply:
that was a hunch